allure-robotframework/test/status/test_step_status.py

```diff
@@ -0,0 +1,37 @@
+import unittest
+import os
+
+from allure_commons_test.report import AllureReport, has_test_case
+from allure_commons_test.result import with_status, has_step
+from hamcrest import assert_that
+
+
+class StepStatus(unittest.TestCase):
+    allure_report = AllureReport(os.path.join('output'))
+
+    def test_passed_with_step(self):
+        assert_that(self.allure_report, has_test_case('Passed Case With Step',
+                                                      with_status('passed'),
+                                                      has_step('BuiltIn.No Operation',
```

Comment:
if 'BuiltIn.No Operation' is a robot framework constant, I would suggest to move it into some file with test constants for all tests.

Reply:
Done